scripts/check_if_test_should_run_v2.ts

```diff
@@ -2,15 +2,21 @@
 // Say the caller is in the celotool dir and wants to check if celotool or protocol dir has changed then
 // the sample usage will be
 // node -r ts-node/register scripts/check_if_test_should_run_v2.ts --dirs packages/protocol,packages/celotool
-// Prints "true" to stdout if the tests should run
-// Prints "false" otherwise
+// Prints "false" if tests shouldn't run, anything else will cause tests to run
```

Comment:
Will "logMessage" avoid this?

IMO this is unnecessarily fragile, could we instead explicitly call `process.exit` to decide whether or not the tests should run?

Reply:
I agree that it's kind of weird but `process.exit(1)` for a script that runs successfully is also not great.